[IMP] empty

arpi-odoo opened the pull request.

arpi-odoo committed:
[FIX] hr: new contract button invisible

arpi-odoo committed:
[IMP] hr_skills: cleanup certification reporting

arpi-odoo committed:
[FIX] hr_holidays: fix traceback leave type

arpi-odoo committed:
[FIX] hr: fix delete version

arpi-odoo committed:
[FIX] hr: fix various bugs

mepe-odoo committed:
[FIX] hr_holidays: fix accrual day selection

arpi-odoo committed:
[IMP] empty

robodoo commented:
This PR targets the un-managed branch odoo-dev/odoo:master-hr-onboarding, it needs to be retargeted before it can be merged.

hazemayman1 committed:
[FIX] hr_holidays: dynamic accrual days

Changed the {day} dropdown to adapt to the selected month. It was previously set to 31 for all months which allowed the user to select the 31st on a 28 or 30 day month for example.

task-5392293
